index.html

```diff
@@ -11,5 +11,9 @@ <h1>p5 sketches</h1>
     <a href="template">template</a>
     <br>
 
+    <a href="dain/index.html">Dain Jung</a>
+    <br>
+    <a href="homework/index.html"> New! </a>
```

Comment:
we need to make sure this is linking to the right place, right now it's looking for a file called "index.html" inside a folder which no longer exists called "homework". also instead of "New!" we need ur name, b/c this is going into a repo w/everyone else's sketches to. so change the line to something like this:
```html
<a href="dain1/index.html"> Dain Jung 1 </a>
```